src/Microsoft.ML.Transforms/NormalizerCatalog.cs

```diff
@@ -51,6 +32,106 @@ public static class NormalizationCatalog
             return new NormalizingEstimator(env, mode, InputOutputColumnPair.ConvertToValueTuples(columns));
         }
 
+        /// <summary>
+        /// Normalize (rescale) the column according to the <see cref="NormalizingEstimator.NormalizationMode.MinMax"/> mode.
```

Comment:
>NormalizingEstimator.NormalizationMode.MinMax [](start = 71, length = 45)

If we don't use it anymore, shall it become internal?

Reply:
Good point...

---
In reply to: [271101091](https://github.com/dotnet/machinelearning/pull/3118#discussion_r271101091) [](ancestors = 271101091)